packages/web/src/hooks/use-owner-action.ts

```diff
@@ -24,7 +24,7 @@ function bytes32(value: string): string {
  */
 
 export const ListId = bytes32('42');
-export const ListName = process.env.NEXT_PUBLIC_KYC_LIST_NAME ?? 'kycPlus';
```

Comment:
This is already in main. No need to do here.

Reply:
If we assume it will always be this one, I can set it hardcoded.
But in case it changes the support is already there, and since a lot of things change fast in this project, I would say to leave it for support, WDYT @zamith and @rafaelcruzazevedo?